packages/utils/src/tests/period/period.test.js

```diff
@@ -108,6 +108,177 @@ context('periodTypes', () => {
+      });
+
+      it('=', () => {
+        expect(comparePeriods('2020W53', '2020W53')).to.equal(0);
+        expect(comparePeriods('2020W52', '20201227')).to.equal(0);
+        expect(comparePeriods('2020W53', '20210103')).to.equal(0);
+      });
+
+      it('<', () => {
+        expect(comparePeriods('2020W52', '2021')).to.be.lessThan(0);
+        expect(comparePeriods('2020W52', '2021Q1')).to.be.lessThan(0);
+        expect(comparePeriods('2020W52', '202101')).to.be.lessThan(0);
+        expect(comparePeriods('2020W52', '2021W01')).to.be.lessThan(0);
+        expect(comparePeriods('2020W52', '20201228')).to.be.lessThan(0);
+        expect(comparePeriods('2020W53', '20210301')).to.be.lessThan(0);
+      });
+    });
+
+    describe('day', () => {
+      it('>', () => {
+        expect(comparePeriods('20200101', '2019')).to.be.greaterThan(0);
+        expect(comparePeriods('20200101', '2019Q4')).to.be.greaterThan(0);
+        expect(comparePeriods('20200101', '201912')).to.be.greaterThan(0);
+        expect(comparePeriods('20191230', '2019W52')).to.be.greaterThan(0);
+        expect(comparePeriods('20200101', '20191231')).to.be.greaterThan(0);
+      });
+
+      it('=', () => {
+        expect(comparePeriods('20201231', '2020')).to.equal(0);
+        expect(comparePeriods('20201231', '202012')).to.equal(0);
+        expect(comparePeriods('20201231', '2020Q4')).to.equal(0);
+        expect(comparePeriods('20201227', '2020W52')).to.equal(0);
+        expect(comparePeriods('20210103', '2020W53')).to.equal(0);
+      });
+
+      it('<', () => {
+        expect(comparePeriods('20201231', '2021')).to.be.lessThan(0);
+        expect(comparePeriods('20201231', '2021Q1')).to.be.lessThan(0);
+        expect(comparePeriods('20201231', '202101')).to.be.lessThan(0);
+        expect(comparePeriods('20201231', '2021W01')).to.be.lessThan(0);
+        expect(comparePeriods('20201231', '20210101')).to.be.lessThan(0);
+      });
+    });
+  });
+
+  describe('isFuturePeriod', () => {
+    const currentDateStub = '2020-12-31T00:00:00Z';
+    let clock;
+
+    beforeEach(() => {
+      clock = sinon.useFakeTimers({ now: new Date(currentDateStub) });
+    });
+
+    after(() => {
+      clock.restore();
+    });
+
+    it('past', () => {
+      expect(isFuturePeriod('2019')).to.be.false;
+      expect(isFuturePeriod('2020Q3')).to.be.false;
+      expect(isFuturePeriod('202011')).to.be.false;
+      expect(isFuturePeriod('2020W52')).to.be.false;
+      expect(isFuturePeriod('20201230')).to.be.false;
+    });
+
+    it('present', () => {
+      expect(isFuturePeriod('2020')).to.be.false;
+      expect(isFuturePeriod('2020Q4')).to.be.false;
+      expect(isFuturePeriod('202012')).to.be.false;
+      expect(isFuturePeriod('20201231')).to.be.false;
+    });
+
+    it('present - week period type', () => {
+      // We need to match the last date of a week
+      clock = sinon.useFakeTimers({ now: new Date('2020-12-27T00:00:00Z') });
```

Comment:
See https://github.com/beyondessential/tupaia/pull/1291/files#diff-341d943defdc3255425fddec1741973b for Igor's util - small difference in format, perhaps you'd like to comment over there if it is better to specify `now` within an object

Reply:
Nice, I didn't know the other syntax! I guess they are equivalent - the object param just allows you to provide more options

Reply:
I'll use the shorter syntax here too